src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 700;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  --color-dynamic: #3d4e61;
```

Comment:
Consider adding a font-family to the body for better typography. Example: font-family: 'Avenir-Book', sans-serif;

Reply:
please let's leave these fonts as they are, otherwise the tests fail

Reply:
Then you can use `font-family: inherit;` for input and its placeholder